pkg/processor/sourceprocessor/README.md

```diff
@@ -2,14 +2,13 @@
 
 The `sourceprocessor` adds `_sourceName` and other tags related to Sumo Logic metadata taxonomy.
 
-It leverages data tagged by `k8sprocessor` and must be after it in the processing chain.
-It has certain expectations on the label names used by `k8sprocessor` which might be configured below.
+It uses data tagged by `k8sprocessor` and must be after it in the processing chain.
```

Comment:
IMHO we don't need it. we can mention that default values depends on them :thinking: